docs/REFERENCE-environment_variables.md

```diff
@@ -24,6 +24,7 @@
 | DEBUG_SCALING                                    | Emit console.log on events related to scaling issues. _Default_: false.                                                                                                                                                                                                                                                                                                                     |
 | DEFAULT_SERVICE                                  | Default SMS service. _Options_: twilio, nexmo, fakeservice.                                                                                                                                                                                                                                                                                                                                 |
 | DEFAULT_ORG                                      | Set only with FIX_ORGLESS. Set to integer organization.id corresponding to the organization you want orgless users to be assigned to.                                                                                                                                                                                                                                                       |
+| <a name="deprecated_ui"></a>DEPRECATED_TEXTERUI  | For a limited time, you can set DPRECATED_TEXTERUI=`GONE_SOON` to restore the old texter ui.  This will be removed in an upcoming release, but is meant to give you time to update training materials and fallback, in case a major regression is found.  You can also just add `?old=1` to the end of the url.                                                                                                                                                                                                                                                   |
```

Comment:
```suggestion
| <a name="deprecated_ui"></a>DEPRECATED_TEXTERUI  | For a limited time, you can set DEPRECATED_TEXTERUI=`GONE_SOON` to restore the old texter ui.  This will be removed in an upcoming release, but is meant to give you time to update training materials and fallback, in case a major regression is found.  You can also just add `?old=1` to the end of the url.                                                                                                                                                                                                                                                   |
```

Spelling correction, but honestly doesn't matter too much :)